### PR TITLE
Implement viscosity calculation buyukurganci_22

### PR DESCRIPTION
**Pull request by felix-r**
PR to implement new viscosity functions as introduced by Büyükurganci et al.

To do before puling:

- [x] Implement new viscosity functions
- [x] Update documentation

### CODE REVIEW

File: dclab/features/emodulus/viscosity.py [hunk 1]
```diff
@@ -133,6 +133,28 @@ def get_viscosity(medium: str = "0.49% MC-PBS",
...
+        The channel width in µm
+    flow_index: float
+        Flow behavior index aka the power law exponent of the shear thinning
+
+    Returns
+    -------
+    shear_rate: float
+        Shear rate in 1/s.
+    """
+    # convert channel width to mm
```
Comment (paulmueller) [inline dclab/features/emodulus/viscosity.py:153]: If the channel width was µm, then this would be a conversion to um...

Comment (felix-r) [reply dclab/features/emodulus/viscosity.py:153]: ?

You have the channel width in µm. You divide by 1000 and get it in mm. In the formula you have [Q] = µl/s = mm^3/s, [L] = mm and $\left[\frac{Q}{L^3}\right]$ = 1/s, what you want in the end. Does this make sense?

Comment (paulmueller) [reply dclab/features/emodulus/viscosity.py:153]: Oh sorry, you are right.

**Comment by felix-r** [comment]
Now that the equations are available on arxiv, I would not write an extensive documentation and rather leave it to you, how to edit the documentation for the new model. Some comments from my side to the notes you have in the docs:

Under the section `Viscosity`:
```
This paragraph is imcomplete:

    - Which of the models should one choose? https://github.com/DC-analysis/dclab/issues/197
    - How does CellCarrier correspond to which concentration of MC in PBS?
```
- I would only chose the old model if you want to compare the elastic modulus with older results. Otherwise, I see no reason to keep using the old model. The new one should be way more accurate.
  - You should highlight in the docs that one should never compare the Young's modulus derived from the old model with those derived from the new model. This is just wrong!
- CellCarrier = 0.49 w% MC-PBS, CellCarrier B = 0.59 w% MC-PBS 

Under the section `Büyükurgancı 2022`: I don't think you need to get into all the things you commented there. If you show the Carreau-Yasuda plot, it should become clear how the viscosity behaves. You can also use Figure 3 from the arxiv paper (2023) to illustrate how the viscosity changes depending on temperature, flow rate and channel size.

Do you still want me to add something to the code?

**Comment by paulmueller** [comment]
Thanks for the comments. I am currently looking into the arxiv paper and will get back to you if I encounter any issues.

File: dclab/features/emodulus/viscosity.py [hunk 2]
```diff
@@ -142,7 +164,28 @@ def get_viscosity_mc_pbs_buyukurganci_2022(
         temperature: float = 23.0):
     """Compute viscosity of MC-PBS according to :cite:`Buyukurganci2022`"""
     check_temperature("'buyukurganci-2022' MC-PBS", temperature, 22, 37)
-    raise NotImplementedError("Model buyukurganci-2022 not implemented yet!")
+    # material constants for temperature behavior of MC dissolved in PBS:
+    alpha = 2.23
```
Comment (paulmueller) [inline dclab/features/emodulus/viscosity.py:168]: According to the arxiv paper https://arxiv.org/abs/2302.01664 `alpha` should be 0.00223, correct?

**Review by paulmueller — CHANGES_REQUESTED** [review]

**Review by paulmueller — APPROVED** [review]

**Pull request merged**